test/widgets/cards_test.dart

```diff
@@ -259,9 +259,6 @@ void main() {
 
       //Rating is shown
       expect(find.byType(CustomRatingBarIndicator), findsOneWidget);
-
-      //Price is shown
-      expect(find.byKey(const Key('price')), findsOneWidget);
```

Comment:
this seems unrelated?

Reply:
the price was before there where the icon was and the price is only shown for preview status

Reply:
ah yes, thanks!